pkg/apis/core/validation/validation_test.go

```diff
@@ -25860,9 +25874,47 @@ func TestValidatePodResize(t *testing.T) {
 			new: mkPod(core.ResourceList{}, getResources("200m", "0", "1Gi", "")),
 			err: "",
 		},
+		{
```

Comment:
nitpick: correct the formatting? `}, {` instead of braces on separate line.

Reply:
Done